src/Components/SmartComponents/SystemsExposedTable/SystemsExposedTable.js

```diff
@@ -126,68 +90,66 @@ const SystemsExposedTable = (props) => {
     }, [dispatch]);
 
     const downloadReport = format => {
-        let params = [props.cve, { ...parameters }];
-        DownloadReport.exec(fetchAffectedSystemsByCVE, params, format, 'systems-exposed', notification => dispatch(
-            addNotification(notification)), () => dispatch(clearNotifications()));
+        DownloadReport.exec(
+            fetchAffectedSystemsByCVE,
+            { id: cveId, ...parameters },
+            format,
+            'systems-exposed',
+            notification => dispatch(addNotification(notification)),
+            () => dispatch(clearNotifications())
+        );
     };
 
+    const getEntities = useGetEntities(
+        APIHelper.getAffectedSystemsByCVE,
+        {
+            createRows: createExposedSystemsRows,
+            id: cveId,
+            setUrlParams
+        }
+    );
+
+    const bulkSelectProps = useBulkSelect({
```

Comment:
Select all shows "0 items"
![0items](https://user-images.githubusercontent.com/8426204/133761790-e0d73f59-f91b-43d7-8d74-88218b44b15a.png)

And it's pretty broken overall
![bulkSelect](https://user-images.githubusercontent.com/8426204/133762633-df9b53f3-377c-4ff8-8e90-17234f336538.gif)

But it was broken (in a different way) even before this PR and [we have a ticket for it](https://issues.redhat.com/browse/VULN-1936) so it's okay